5-network/06-fetch-api/article.md

```diff
@@ -213,12 +213,12 @@ window.onunload = function() {
 };
 ```
 
-Normally, when a document is unloaded, all associated network requests are aborted. But `keepalive` option tells the browser to perform the request in background, even after it leaves the page. So this option is essential for our request to succeed.
+Normally, when a document is unloaded, all associated network requests are aborted. But the `keepalive` option tells the browser to perform the request in the background, even after it leaves the page. So this option is essential for our request to succeed.
 
 It has a few limitations:
 
 - We can't send megabytes: the body limit for `keepalive` requests is 64KB.
     - If we need to gather a lot of statistics about the visit, we should send it out regularly in packets, so that there won't be a lot left for the last `onunload` request.
-    - This limit applies to all `keepalive` requests together. In other words, we can perform multiple `keepalive` requests in parallel, but the sum of their body lengths should not exceed 64KB.
+    - This limit applies to all `keepalive` requests. In other words, we can perform multiple `keepalive` requests in parallel, but the sum of their body lengths should not exceed 64KB.
```

Comment:
I'd like to emphasize that the limit applies to all of them - not to any of them, but all together. What's wrong here? Should we rephrase it?

Reply:
> I'd like to emphasize that the limit applies to all of them - not to any of them, but all together. What's wrong here? Should we rephrase it?

To my subjective understanding the "together" could be omitted. You already say "all keepalive requests". Up to you :)

Reply:
Please leave it be, for someone less smart than you are =)

Reply:
Restored the "together" :)